store/src/java/com/zimbra/cs/ephemeral/EphemeralStore.java

```diff
@@ -302,7 +324,17 @@ public static Factory getFactory() throws ServiceException {
     }
 
     public static Factory getFactory(String backendName) {
-        String factoryClassName = factories.get(backendName);
+        return getFactory(backendName, false);
```

Comment:
Ah good catch - this causes an NPE if the destination set from zmmigrateattrs is invalid.